.gitignore

```diff
@@ -1,5 +1,13 @@
 .DS_Store
+.envrc
 *~
 *.autosave
 CMakeLists.txt.user*
 build/
+
```

Comment:
does this work as those file/folders in in the js subfolder? You can put a `.gitignore` in `js/` also.

Reply:
Yes, this works.